projects/plugins/crm/src/automation/commons/triggers/events/class-event-deleted.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Jetpack CRM Automation Event_Deleted trigger.
+ *
+ * @package automattic/jetpack-crm
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Triggers;
+
+use Automattic\Jetpack\CRM\Automation\Base_Trigger;
+
+/**
+ * Adds the Event_Deleted class.
+ */
+class Event_Deleted extends Base_Trigger {
```

Comment:
Another thing that we generally need to clean up in automations, but WP code standards wants us to add a `@since` tag here.

https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#2-classes
```suggestion
/**
 * Adds the Event_Deleted class.
 * 
 * @since $$next-version$$
 */
class Event_Deleted extends Base_Trigger {
```